serve_event_listener/http_client/client.py

```diff
@@ -19,25 +20,30 @@ def _request(
     params: Optional[Mapping[str, Any]] = None,
     json: Optional[Mapping[str, Any]] = None,
     headers: Optional[Mapping[str, str]] = None,
-    verify: bool = True,
+    verify: Optional[VerifyType] = None,
     timeout: Timeout = (3.05, 20.0),
     backoff_seconds=(1, 2, 4),
     token_fetcher: Optional[Callable[[], str]] = None,  # optional
     auth_scheme: str = "Token",  # can easily change to Bearer in the future
     sleep_fn: Callable[[float], None] = time.sleep,  # overridable in tests
+    **request_kwargs,  # pass-through for requests.Session.request
 ) -> Optional[requests.Response]:
     merged_headers = {**(session.headers or {}), **(headers or {})}
 
     if not backoff_seconds:
         raise ValueError("backoff_seconds must contain at least one delay value")
 
-    # If token_fetcher is provided but Authorization is missing, fetch once
+    # if token_fetcher is provided but Authorization is missing, fetch once
     refreshed = False
     if token_fetcher and "Authorization" not in merged_headers:
         tok = token_fetcher()
         if tok:
             merged_headers["Authorization"] = f"{auth_scheme} {tok}"
 
+    # if verify not supplied, inherit from the Session (default True if absent)
+    if verify is None:
+        verify = getattr(session, "verify", True)
```

Comment:
is there a need to use getattr here?

Reply:
I see
I would've preferred not relying on getattr that much, but let's keep it that way for now